picard/tagger.py

```diff
@@ -595,6 +533,20 @@ def handle_command_lookup_cd(self, argstring):
             partial(self._lookup_disc, disc),
             traceback=self._debug)
 
+    def handle_command_pause(self, argstring):
+        arg = argstring.strip()
+        if arg:
+            try:
+                _delay = float(arg)
```

Comment:
Is there a specific reason to use `_delay` over `delay` here?

Reply:
No reason at all.  I've made the change.

In addition, I've removed the 30 second command timeout.  After thinking about it I couldn't come up with a single situation when it might be needed, but it could cause a long running command to show as being completed prematurely causing problems with the sequencing.  Some examples could be: scanning or looking up a large number of clusters; loading a release with a lot of coverart images; or fingerprinting a large number of files.